### PR TITLE
DCP 2 Migration

### PR DESCRIPTION
**Pull request by paulrobinson**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -45,9 +45,9 @@ gem 'docker-api', :require => 'docker'
 #    gem 'aweplug', github: '<github_id>/aweplug', :branch => '<branch_name>'
 #
 # From a location on your disk:
-#    gem 'aweplug', path: '<path_to_aweplug_on_your_disk>'
 #
-gem 'aweplug', github: 'awestruct/aweplug'
+# gem 'aweplug', github: 'awestruct/aweplug'
+gem 'aweplug', github: 'wesbos/aweplug', :branch => 'dcp2-updates'
```
Comment (paulrobinson) [inline Gemfile]: We need to merge this change into Aweplug master

**Comment by paulrobinson** [comment]
@wesbos any idea why the related videos stuff isn't working? See: http://it-developers-pr.stage.redhat.com/pr/690/build/452/video/youtube/Cl3SiavsBF4/

**Comment by wesbos** [comment]
That video doesn't have any products associated with it, so we are unable to find related videos.

It's still working:

http://it-developers-pr.stage.redhat.com/pr/690/build/452/video/youtube/2lg8rbz9buU/

**Comment by paulrobinson** [comment]
I see the problem now. The EAP learn page is not filtering by project initially. See: http://it-developers-pr.stage.redhat.com/pr/690/build/452/products/eap/learn/

However, once a filter is applied, the project filter is applied. See: http://it-developers-pr.stage.redhat.com/pr/690/build/452/products/eap/learn/#/sys_type=video&project=eap&randomize=false&query=

However, this then results in a broken query being sent to the DCP:

http://dcp2-searchisko.rhcloud.com/v2/rest/search/developer_materials?newFirst=true&project=eap&size10=true&sys_type=video

I'm investigating what's broken here.

**Comment by paulrobinson** [comment]
@wesbos any ideas why the learn page doesn't filter by 'project' when first visited?

**Comment by wesbos** [comment]
Not off the top of my head by I can dig into it if you'd like

**Comment by paulrobinson** [comment]
@wesbos do you have time to look at this today? It doesn't look like I will have time, and was hoping to merge the DCP code first thing tomorrow.

**Comment by paulrobinson** [comment]
I'll get it rebased incase you need to build it

**Comment by wesbos** [comment]
@paulrobinson LMK when you rebase and I'll dig in

**Comment by paulrobinson** [comment]
@wesbos rebased. Appreciated!

**Comment by wesbos** [comment]
I've sent you a PR fixing this

**Comment by paulrobinson** [comment]
Another problem: http://it-developers-pr.stage.redhat.com/pr/690/build/502/video/vimeo/131199128/

Notice the related videos are not displayed, and the following error is in the JS console:

      Uncaught TypeError: video.fields.sys_url_view.replace is not a function

**Comment by paulrobinson** [comment]
@wesbos Also the '...' is clickable on http://it-developers-pr.stage.redhat.com/pr/690/build/502/resources/. It takes you to a broken version of the page. In production today that link is not clickable.

**Comment by paulrobinson** [comment]
Otherwise it all seems to be working.

**Comment by paulrobinson** [comment]
@wesbos are you able to take a look at those two issues today?

**Comment by wesbos** [comment]
Yep - doing it now

**Pull request merged**